account_asset_management/views/account_move.xml

```diff
@@ -22,29 +51,25 @@
                 </button>
             </xpath>
             <xpath
-                expr="//field[@name='invoice_line_ids']/tree/field[@name='quantity']"
-                position="before"
+                expr="//field[@name='invoice_line_ids']/tree/field[@name='asset_profile_id']"
+                position="after"
             >
-                <field
-                    name="asset_profile_id"
-                    domain="[('company_id','=', parent.company_id)]"
-                    attrs="{'invisible': [('parent.type', 'not in', ('in_invoice', 'in_refund'))]}"
-                />
                 <field
                     name="asset_id"
-                    attrs="{'invisible': [('parent.type', 'not in', ('out_invoice', 'out_refund'))]}"
+                    attrs="{'column_invisible': [('parent.type', 'not in', ('out_invoice', 'out_refund'))]}"
                     groups="account.group_account_manager"
```

Comment:
No need of this group, as it's in header.

Reply:
The idea was to remove it, thanks